src/components/CreateUserCreationRequestInternalEmployeeForm/CreateUserCreationRequestInternalEmployeeForm.tsx

```diff
@@ -148,8 +148,7 @@ export const CreateUserCreationRequestInternalEmployeeForm = ({
                     <Text weight="bold" color={gray8}>
                         {tr('Organization:')}
                     </Text>
-
-                    <OrganizationUnitComboBox onChange={onOrganizationChange} />
+                    <OrganizationUnitComboBox onChange={onOrganizationChange} searchType="internal" />
```

Comment:
searchType='external' is needed in ExternalEmployeeForm

Reply:
Sorry, it should be a typo error